hazelcast-enterprise-kubernetes/mvnw/mvnw

```diff
@@ -0,0 +1,236 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
```

Comment:
I am not sure what this script is doing but I think that is something we need to maintain over time. It is probably complete copy of a script from somewhere else.

Reply:
Yes from https://github.com/takari/maven-wrapper
It's a good idea to have this installed from the Dockerfile itself.